### PR TITLE
docs: Google Compute Engine

### PR DESCRIPTION
**Pull request by pohly**
It is currently possible to use PMEM on some GCE machines, but
installing Kubernetes and PMEM-CSI is only possible with modified
images and scripts.

### CODE REVIEW

File: examples/gce.md [hunk 1]
```diff
@@ -0,0 +1,467 @@
+# Google Cloud Engine
+
+Google Cloud Compute Engine (GCE) offers VMs with Intel Optane
+Persistent Memory as part of an alpha program.  Request to get
```
Comment (ss7pro) [inline examples/gce.md]: Optane DC Persistent Memory :P

**Comment by fproulx-eoscanada** [comment]
👍 ❤️

File: examples/gce.md (hunk 1 above)
Comment (pohly) [reply examples/gce.md]: Even better: Intel® Optane™ DC persistent memory

**Comment by pohly** [comment]
Updated, @avalluri please merge.

**Pull request merged**